modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -346,57 +365,60 @@ a|
+|sonarqube |https://www.npmjs.com/package/@janus-idp/backstage-scaffolder-backend-module-sonarqube[@janus-idp/backstage-scaffolder-backend-module-sonarqube] |backend-plugin-module |1.4.10 |Red Hat Tech Preview
 |./dynamic-plugins/dist/janus-idp-backstage-scaffolder-backend-module-sonarqube-dynamic
-| |Disabled
-
-|Tech Radar |Frontend |@backstage/plugin-tech-radar |A Backstage plugin
-that lets you display a Tech Radar for your organization |0.6.13 |Community Support
-|./dynamic-plugins/dist/backstage-plugin-tech-radar | |Disabled
+a|
+|Disabled
 
-|Techdocs |Frontend |@backstage/plugin-techdocs |The Backstage plugin
-that renders technical documentation for your components |1.10.0
-|Production |./dynamic-plugins/dist/backstage-plugin-techdocs |
+|techdocs |https://www.npmjs.com/package/@backstage/plugin-techdocs[@backstage/plugin-techdocs] |frontend-plugin |1.10.4 |Production
+|./dynamic-plugins/dist/backstage-plugin-techdocs-dynamic
+a|
 |Enabled
 
-|Techdocs |Backend |@backstage/plugin-techdocs-backend |The Backstage
-backend plugin that renders technical documentation for your components
-|1.9.6 |Production
-|./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic a|
-`TECHDOCS_BUILDER_TYPE`
-
-`TECHDOCS_GENERATOR_TYPE`
-
-`TECHDOCS_PUBLISHER_TYPE`
-
-`BUCKET_NAME`
-
-`BUCKET_REGION_VAULT`
+|techdocs |https://www.npmjs.com/package/@backstage/plugin-techdocs-backend[@backstage/plugin-techdocs-backend] |backend-plugin |1.10.4 |Production
+|./dynamic-plugins/dist/backstage-plugin-techdocs-backend-dynamic
+a|
+|Enabled
 
-`BUCKET_URL`
+|tech-radar |https://www.npmjs.com/package/@backstage/plugin-tech-radar[@backstage/plugin-tech-radar] |frontend-plugin |0.7.4 |Community Support
+|./dynamic-plugins/dist/backstage-plugin-tech-radar-dynamic
+a|
+|Disabled
 
-`AWS_ACCESS_KEY_ID`
+|tekton |https://www.npmjs.com/package/@janus-idp/backstage-plugin-tekton[@janus-idp/backstage-plugin-tekton] |frontend-plugin |3.7.5 |Production
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton-dynamic
+a|
+|Disabled
 
-`AWS_SECRET_ACCESS_KEY`
+|topology |https://www.npmjs.com/package/@janus-idp/backstage-plugin-topology[@janus-idp/backstage-plugin-topology] |frontend-plugin |1.21.7 |Production
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-topology-dynamic
+a|
+|Disabled
 
+|utils |https://www.npmjs.com/package/@roadiehq/scaffolder-backend-module-utils[@roadiehq/scaffolder-backend-module-utils] |backend-plugin-module |1.15.3 |Community Support
+|./dynamic-plugins/dist/roadiehq-scaffolder-backend-module-utils-dynamic
+a|
 |Enabled
 
-|Tekton |Frontend |@janus-idp/backstage-plugin-tekton |The Tekton plugin
-enables you to visualize the PipelineRun resources available on the
-Kubernetes cluster. |3.5.12 |Production
-|./dynamic-plugins/dist/janus-idp-backstage-plugin-tekton | |Disabled
 |===
+[NOTE]
+====
+* To configure Keycloak, see xref:rhdh-keycloak_{context}[Installation and configuration of Keycloak].
```

Comment:
Why are we calling out Keycloak configuration here? Is it any different to the other plugins that we have documented and do we need to do likewise for those e.g. Nexus or Tekton.

Reply:
Yes. Because it was called out in the 1.1 docs and I don't have a way to inject that into the dynamically-created content. So  I figured having it underneath was good enough for horeshoes and handgrenades.

Reply:
How it looks in the 1.1 docs:

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/227597/bd0289c5-ebc5-4863-9aed-7484955c7972)

I'd suggest making this link way more user friendly in a note below the table.

Reply:
@nickboldt Ok, I see your point about Keycloak and I'm ok with calling it out like this for 1.2.

I also see that we are now linking each plugin to the Backstage documentation which we did not do in previous versions. The Backstage docs have different procedures for enabling plugins, so will this not possibly confuse our customers? 

I'd suggest removing these links, if we do provide links in the table I think they should point to the RHDH docs where they exist e.g. Keycloak in 1.1

Reply:
the links are to the public npmjs.com listing for the node packages consumed by both Backstage and RHDH. I don't see why this is a bad thing, as it provides MORE information than what was in 1.1

The whole point of moving the keycloak link out of the table was that I can't just have a single rule for a single plugin to link to another section in the doc, unless we can ensure that the section IDs for those plugin-specific pages match the IDs of the plugins themselves; I could then look for filenames that match and only generate the link if the content exists elsewhere in the plugin config guide

Reply:
links removed.